src/utils/fetchRockets.js

```diff
@@ -0,0 +1,11 @@
+/** @format */
+
+import { ROCKETS_URL } from './ApiData';
+
+const fetchRockets = async () => {
```

Comment:
[OPTIONAL] I would suggest adding a screenshot of your project to your readme file. Also, do not forget to include MIT license as well.